develop.txt

```diff
@@ -1,10 +1,5 @@
 coverage>=5.5
-flake8>=4
-nose>=1.3.7
 pytest>=6.2.2
 pytest-cov>=2.11.1
-pytest-pep8>=1.0.6
 pytest-emoji>=0.2.0
-pytest-flake8>=1.0.7
-wemake-python-styleguide>=0.15.2
```

Comment:
OK, I would propose we replace the outgoing development dependencies with the the following:

```
black
isort
pytest-pycodestyle
pytest-pydocstyle
```

Reply:
The complete shift to black in the CI and applying to the code should be done after though, else the merging of the coming PRs are going to be _complicated_